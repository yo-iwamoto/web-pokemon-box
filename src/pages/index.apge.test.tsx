import Page from './index.page';
import { render } from '@testing-library/react';

describe('/ Page Component', () => {
  it('renders correctly', () => {
    const { getByRole, asFragment } = render(<Page />);

    expect(
      getByRole('heading', {
        name: 'Hello world',
      })
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
