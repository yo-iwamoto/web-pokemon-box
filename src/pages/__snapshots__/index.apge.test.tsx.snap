// Vitest Snapshot v1

exports[`/ Page Component > renders correctly 1`] = `
<DocumentFragment>
  <h1>
    Hello world
  </h1>
</DocumentFragment>
`;
